### DISCUSSION TITLE
How can I configure local feeds?

### DISCUSSION THREAD

**Discussion by slalaurette** [martinrotter/rssguard#1008] [Q&A]
I have some local feeds that are in my computer in the form of .xml files. I have these after importing from QuiteRSS. They work for fetching the articles, but, when trying to open them, they don't work, because RSS Guard prepends "file:///" to the URL. 

How can I solve this?

**Comment by martinrotter** [comment]
Hi. Can you give some screenshot of what exactly happens?

**Reply by slalaurette** [reply]
Yes! 

QuiteRSS showing the properties for the feed: https://i.imgur.com/ZBuG5kK.jpg
Individual item open in QuiteRSS: https://i.imgur.com/1z8KqaE.jpg

RSS Guard showing the properties for the same feed: https://i.imgur.com/LgUSihQ.jpg
Individual item (failing to) open in RSS Guard: https://i.imgur.com/3E1k7q0.jpg

**Comment by martinrotter** [comment]
Provide the XML feed file please

Dne po 10. 7. 2023 22:04 uživatel slalaurette ***@***.***>
napsal:

**Reply by slalaurette** [reply]
Here it is! 
Zipped because it's an unsupported type. 
[ConcursosxEmail.zip](https://github.com/martinrotter/rssguard/files/12008749/ConcursosxEmail.zip)

**Comment by martinrotter** [comment]
Hi. I checked your situation. In RSS Guard there are several related behaviors to your problem and they are very complex. I made some cosmetic fixes for other problems I found during trying your feed.

There cannot be any universal fix to make. RSS Guard automatically tries to resolve relative URLs of articles when feed is fetched and usually it works great for feeds posted in regular online URLs. In your case you need to apply this article filter below:

```js
function filterMessage() {
  msg.url = "https://www.escritores.org/" + msg.url.replace("file:///", "");

  return MessageObject.Accept;
}
```

Simply go to "Article filters" dialog, create new filter and name it and paste this code into it. Then add checkbox to your feed and **clean existing articles** from the feed (or remove and re-add the feed beforehand). Then it should work.

![image](https://github.com/martinrotter/rssguard/assets/1255302/ec9a1c29-a11d-4e4b-8202-ffc725b527cb)

**Reply by slalaurette** [reply]
Thank you so much! I will look into it.
